tools/icqsol_add_texture/icqsol_add_texture.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+import argparse
+import shutil
+from pkg_resources import resource_filename
```

Comment:
@pletzer with Galaxy tools, the data file is uploaded to a history and selected by the Galaxy tool from the history and passed as a file path to the underlying package code.